api-extractor.json

```diff
@@ -45,7 +45,7 @@
    *
    * SUPPORTED TOKENS: <projectFolder>, <packageName>, <unscopedPackageName>
    */
-  "mainEntryPointFilePath": "<projectFolder>/lib/esm/index.d.ts",
+  "mainEntryPointFilePath": "<projectFolder>/lib/esm/src/index.d.ts",
```

Comment:
For some reason something related to this change caused build errors where it could not find the path. <projectFolder>/lib/esm/index.d.ts. After searching for this file, I found it at "<projectFolder>/lib/esm/src/index.d.ts